Modules/Core/SpatialObjects/include/itkSpatialObjectProperty.h

```diff
@@ -55,6 +55,12 @@ class ITKSpatialObjects_EXPORT SpatialObjectProperty
     m_Color = color;
   }
 
+  ColorType
+  Color()
+  {
+    return m_Color;
+  }
```

Comment:
Please declare Color() `const` (just like the other new member functions proposed by this pull request).